src/dcm/agent/cmd/configure.py

```diff
@@ -33,12 +33,12 @@
 import dcm.agent.utils as agent_utils
 
 
+ip_addr = '127.0.0.1'
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this IP "127.0.0.1" address configurable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=python%3AS1313)